src/main/launcher/main.c

```diff
@@ -143,6 +143,7 @@ int main(int argc, const char **argv)
     /* Read command line */
 
     options_init(&options);
+    options_read_early_cmdline(&options, argc, argv);
```

Comment:
Might be good to have reasoning in the commit why this is required.

Reply:
Sure.